### PR TITLE
Accessibility branch

### PR DESCRIPTION
**Pull request by beyk**
### Page titles : 
https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/index.html#L9
 we changed it to My Portfolio, It's so descriptive
### Image text alternatives
 For example: 
https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/index.html#L32

https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/index.html#L50-L62

https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/index.html#L72-L73

https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/index.html#L101-L102

https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/index.html#L101-L102

https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/index.html#L157-L158

https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/index.html#L198-L211

https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/index.html#L223-L244
 We add descriptive text alternatives to our images
### Text headings
No issues
### Color contrast:
 We changed some sections color to be more accessabilty:
as you see we changed ul text color in works cards:
https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/styles/main.css#L190-L195
 and also we changed lets connect link color:
https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/styles/main.css#L270-L276

### Resize
No issues

### Interaction
 We changed contact form input section focus mode color:
https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/styles/main.css#L346-L349
and textarea focus color:
https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/styles/main.css#L364-L366
 and we add submit-btn focus color:
https://github.com/beyk/Portfolio-mobile-version/blob/97d4d5396e221446ebfec53a54883bbe0b3b5a32/styles/main.css#L402-L404

### Moving content
No issues

### Multimedia
No issues

### The basic structure of the page
No issues

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<link rel="stylesheet" href="styles/main.css" />
 		<link rel="stylesheet" href="styles/desk-style.css" />
-		<title>Mobile portfoilo version</title>
+		<title>My portfoilo </title>
```
Comment (medaminedev66) [inline index.html]: - [ ] The page title is not descriptive, please see the examples below and kindly fix that:
- Poor titles:
   - _Microverse project | Your portfolio._
   - _Portifolio project_
- Better page titles:
   - _Mark's portifolio_
   - _Mark full-stack webdeveloper_

File: index.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
 		</script>
 	</head>
 	<body>
```
Comment (medaminedev66) [inline index.html:21]: - [ ] The accessibility of your portfolio looks great 😃, but we still have some issues when we zoom the font. Let's make sure that Text, images, and other content do not overlap.
- Your design after zooming the font :
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @beyk and @KanoCode, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**